README.md

```diff
@@ -1,17 +1,44 @@
-# Google CloudEvents – Python
+# Google Events Library for Python
 
-This repository contains types for CloudEvents issued by Google.
+This library provides classes of common event types on Google Cloud Platform
+(GCP). At this moment the following types are available:
```

Comment:
How is this table generated? It doesn't have links to the actual source code.

Maybe we can add this in a different PR when we have a way to generate it?

Reply:
The README file is now generated in the script.